ministryofjustice/analytics-platform-control-panel
# ANPL-1603 Fixed the bug during registering an app

## :memo: Summary
This PR is to fix issue in #ANPL-1603

Merging this PR will have the following side-effects:
- No

## :mag: What should the reviewer concentrate on?
- code changes

## :technologist: How should the reviewer test these changes?
- login as superuser
- click `home` from top menu
- click `list all apps`
- choose `register an app` with the option of creating a bucket 
- confirm an app will be registered in control panel and the bucket will be created 

## :books: Documentation status
